src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfo.Cache.cs

```diff
@@ -73,7 +73,6 @@ internal Dictionary<string, JsonPropertyInfo> PropertyIndex
         /// <summary>
         /// Defines the core property lookup logic for a given unescaped UTF-8 encoded property name.
         /// </summary>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Seems like this ends up inlining a lot of code (+ some of the calls it contains are marked as AggressiveInlining). I'll run benchmarks.

Reply:
Some of the AggressiveInlining methods are there to prevent deeper stacks in recursive calls, but I think remove this specific one is fine. I haven't done this myself so far given how notoriously sensitive performance is to changing these files.